api/src/services/auth/authmanagement/authmanagement.test.ts

```diff
@@ -44,7 +59,46 @@ describe("'authManagement' service", () => {
      * We don't care about hours in this case.
      */
     expect.assertions(1)
-    const user = await app.service('user').create(credentials)
+    const user = await app.service(SERVICES.CORE_USER).create({ ...credentials })
```

Comment:
```suggestion
    const user = await app.service(SERVICES.CORE_USER).create(credentials)
```

Reply:
Well, there is a bug I encounter and `credentials` was mutated.
So I decide to spread properties to avoid mutation of `credentials`.
Some properties had been added, this was breaking my tests.

Reply:
ok, 
Isn't it simpler and more maintainable to isolate the test in question and always have the same shape for the others?